src/lib/components/explorer/export/Summary.svelte

```diff
@@ -1,30 +1,37 @@
-<script>
-  import { filters, totalParticipants } from '$lib/stores/Filter';
+<script lang="ts">
   import ExportStore from '$lib/stores/Export';
+  import { filters, totalParticipants } from '$lib/stores/Filter';
   let { exports } = ExportStore;
 
   $: participantsCount = $totalParticipants;
   $: variablesCount = $filters.length + $exports.length;
-  $: dataPoints = participantsCount * variablesCount;
+  $: dataPoints = typeof participantsCount === 'number' ? participantsCount * variablesCount : 0;
```

Comment:
Why 0 and not parse the int

Reply:
Is the string numeric? I assumed it was a message or something. If it is a number why is it a string?